### ISSUE TITLE
Bisenetv2

### ISSUE THREAD

**Issue by huang229** [CoinCheung/BiSeNet#52]
Bisenetv2 has come out, and my reproduction accuracy is only 66.7%. The author's paper is 73.6%. This bisenetv2 has better real-time performance. Would God be interested in reproducing it?

**Comment by 18022443868** [comment]
the same question

**Comment by dronefreak** [comment]
Hi @huang229 

Please make sure that your training environment is similar to what has been mentioned in this repo. I have reproduced the results successfully. An overview of your training setup might help in debugging the issue better.

**Comment by huang229** [comment]
@dronefreak   I'm talking about bisenetv2, not bisenetv1. Friend, are you sure you have reproduced the effect of bisenetv2? The channel number of Ge module in the network structure in the paper is not very clear. How do you set it?

**Comment by dronefreak** [comment]
Ahh my bad. I did not read that properly. But indeed, BiseNEtv2 has some claims that I am not yet sure of. I am in the process of implementing the model myself to see how effective their modules are.

**Comment by huang229** [comment]
@dronefreak Some modules of bisenetv2 are not explained in detail. For example, the number of channels in the Ge module. I implement according to my own understanding, and the result accuracy is only 66.7% . but in the paper 73.36% (I'm wrong about the accuracy value, No 73.6% ）。

**Comment by 18022443868** [comment]
Wasn't there a number of channels in the original paper

**Comment by huang229** [comment]
@18022443868  In this paper, the channel number of Ge module branches in Figure 5 (c) is not clear.

**Comment by 18022443868** [comment]
> @ 18022443868 本文不清楚图5（c）中Ge模块分支的通道号。

128？

**Comment by 18022443868** [comment]
> @ 18022443868 本文不清楚图5（c）中Ge模块分支的通道号。

You're right, wait for the original author to open source

**Comment by huang229** [comment]
@CoinCheung Great God, are you interested in this paper?

**Comment by 18022443868** [comment]
Bisenetv2 available in embedded real time?

**Comment by huang229** [comment]
@18022443868 I think it's hard. In addition, I train slower than V1, I don't know why?

**Comment by 18022443868** [comment]
Ok, thanks for your solution, open source and test again

> @ 18022443868我认为很难。另外，我的训练速度比V1慢，我不知道为什么吗？
Ok, thanks for your solution, open source and test again

**Comment by dronefreak** [comment]
@huang229 the training is indeed slower than V1, for the same crop-size, batchsize and iterations. Perhaps the authors should open-source their codes, but I am not sure when would that happen. I will update this comment once the training is complete, with mIOU I achieve.

**Comment by huang229** [comment]
@dronefreak Good luck, hope you can succeed, and then share your experience.

**Comment by CoinCheung** [comment]
I will have a try. After going through the paper, there are some details that I feel not very clear, so it maybe not so easy to reproduce it. I will update if I can reach the accuracy.

**Comment by 18022443868** [comment]
> 我会尝试的。阅读完本文后，有些细节我不太清楚，因此复制起来可能并不容易。我会更新，如果我能达到的准确性。
Thank god for coming back. Sit back！！！

**Comment by huang229** [comment]
Ha ha, wait for the good news from the God. I hope you can succeed. Your success will be a great help to us.

**Comment by MaybeShewill-CV** [comment]
@CoinCheung My bisenetv2 also can not achieve 73.4 in cityscapes val dataset ==!. So curious about the origin author's implementation:)

**Comment by umairanis03** [comment]
Hi @CoinCheung @dronefreak 

I am unable to download pre-trained model using pan-Baidu. Can you provide me it's google drive link?
